test/functional/index.js

```diff
@@ -1,5 +1,5 @@
 /* global after, describe, before, beforeEach, expect, it */
-import { h, render } from 'preact' /** @jsx h */
+import { createElement, render } from 'preact' /** @jsx createElement */
```

Comment:
If we do, the compilers will still work (possibly not for the React bundle), but Standard.js will complain that it's an unused variable.